alcs-frontend/src/app/features/application/application-meeting/application-meeting-dialog/application-meeting-dialog.component.html

```diff
@@ -6,54 +6,54 @@ <h2 class="card-title">Edit {{ meetingType.label }}</h2>
     <div class="flex-container">
       <mat-form-field appearance="outline" class="flex-child">
         <mat-label>{{ data.meetingType.code === 'AM' ? 'Meeting' : 'Visit' }} Requested</mat-label>
-        <mtx-datetimepicker #meetingStartDate type="date"></mtx-datetimepicker>
         <input
           matInput
-          [mtxDatetimepicker]="meetingStartDate"
+          (click)="meetingStartDatePicker.open()"
+          [matDatepicker]="meetingStartDatePicker"
           formControlName="meetingStartDate"
-          required
-          (dateChange)="onChange()"
+          name="meetingStartDate"
         />
-        <mtx-datetimepicker-toggle [for]="meetingStartDate" matSuffix></mtx-datetimepicker-toggle>
```

Comment:
Are we still using the mtx datepicker anywhere? Can we remove the mtx library?

Reply:
I'll check and remove it in next MR